### DISCUSSION TITLE
个人简介的section序号和附录的序号连在一起

### DISCUSSION THREAD

**Discussion by YiweiSu** [note286/xduts#183] [Q&A] [answered]
![xduts_discussion](https://github.com/note286/xduts/assets/50658576/ce9904b9-a485-441c-a821-06a5d77cd571)
如图，附录\chapter{}下有两个section，个人简介的section序号从3开始了。
尝试过，如果去掉附录，个人简介的section序号就正常从1开始。请问大佬这种情况如何解决

**Comment by note286** [comment, marked as answer]
你在`bio.tex`文件顶部添加

```latex
\setcounter{section}{0}
```

来暂时解决吧。

**Reply by YiweiSu** [reply]
谢谢大佬~

**Comment by note286** [comment]
https://github.com/note286/xduts/commit/8405da4d80a66d6e22745ba455da70b8e76ae8a7 已修复。
